static/src/javascripts/projects/common/modules/ui/subscription-banner-template.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+
+import marque36icon from 'svgs/icon/marque-36.svg';
+import {
+    makeHtml as makeFirstPvConsentHtml,
+    hasUnsetAdChoices as firstPvHasUnsetAdChoices,
+} from 'common/modules/ui/first-pv-consent-banner';
+
+const subsciptionBannerTemplate = (
+    subscriptionUrl: string,
+    signInUrl: string
+): string => `
+<div id="js-subscription-banner-site-message" class="site-message--subscription-banner">
+    <div class="site-message--subscription-banner__inner">
+        <h3 class="site-message--subscription-banner__title">
+            A beautiful way to read it <br>
+            A powerful way <br class="temp-mobile-break" /> to fund it
+        </h3>
+
+
+            <div class="site-message--subscription-banner__description">
+                <p>Two innovative apps and ad-free reading on theguardian.com. The complete digital experience from The Guardian</p>
+            </div>
+            <div class="site-message--packshot-container">
+                <img srcset="https://media.guim.co.uk/28370863b7bb19c5e8e0dc50fe871d4cca99778b/0_0_1894_1156/500.png" src="https://media.guim.co.uk/28370863b7bb19c5e8e0dc50fe871d4cca99778b/0_0_1894_1156/500.png" >
+            </div>
+
+
+        <div class="site-message--subscription-banner__cta-container">
+            <a
+                id="js-site-message--subscription-banner__cta"
+                data-link-name="subscription-banner : to-sign-in"
+                data-link-name="subscription-banner : success"
```

Comment:
why do we have 2 data-link-name attributes?

Reply:
removed the data link attributes they were from the email sign in banner I used as a template